src/cart/crt_hlc.c

```diff
@@ -28,12 +28,25 @@
 #include <gurt/atomic.h>
 #include <time.h>
 
-#define CRT_HLC_MASK 0xFFFFULL
+/**
+ * HLC timestamp unit (given in the HLC timestamp value for 1 ns) (i.e.,
+ * 1/16 ns, offering a 36-year range)
+ */
+#define CRT_HLC_NSEC 16ULL
+
+/**
+ * HLC start time (given in the Unix time for 2021-01-01 00:00:00 +0000 UTC in
+ * seconds) (i.e., together with CRT_HLC_NSEC, offering a range of [2021, 2057])
+ */
+#define CRT_HLC_START 1609459200ULL
```

Comment:
maybe change name to CRT_HLC_START_SEC  for clarity

Reply:
OK.